system-tests/Systemtests

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+pushd `dirname $0` > /dev/null
+SCRIPTPATH=`pwd`
```

Comment:
Let's sue `dirname` to get the script path. It might also makes sense to add `readlink` similar to our git hooks to properly deal with symlinks.  

```shell
SCRIPT_PATH="$(dirname "$0")/$(dirname "$(readlink "$0")")"
```